changelogs/fragments/124-vmware-migrate-vmk-to-std-switch.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- to also allow migration from a VMware vSphere Distrubuted Switch to a ESXi Standard Switch
```

Comment:
I suggest instead:
`- vmware_migrate_vmk - allow migration from a VMware vSphere Distrubuted Switch to a ESXi Standard Switch`

When we build the final changelog, we flatten all the fragments. By prefixing the line with the module name, we improve the readability.